### ISSUE TITLE
Link to Microsoft Store app

### ISSUE THREAD

**Issue by zooba** [python/pythondotorg#1370]
Now that Python is available in the Microsoft Store, it would be great to have 

Information about badges is available at: https://developer.microsoft.com/en-us/store/badges

A zip file containing current badges is available [here](https://go.microsoft.com/fwlink/p/?LinkId=529771), but the best one is available from their CDN at [this url](https://assets.windowsphone.com/85864462-9c82-451e-9355-a3d5f874397a/English_get-it-from-MS_InvariantCulture_Default.svg).

The link to the store should either be:
```
ms-windows-store://pdp/?productid=9NJ46SX7X90P
```
or
```
https://www.microsoft.com/store/apps/9NJ46SX7X90P?ocid=badge
```

The first URL only works on Windows 10, but it will go directly to the store app. So if we're confident in the UA sniffing then it's a better option. The second link will work on any machine.

I think it'd be best to appear on this obvious Download section (rough mockup below) and we'll add it to the Downloads page as a text link next time we do an update.
![image](https://user-images.githubusercontent.com/1693688/50530294-2ce4eb80-0b50-11e9-88f9-bd916b4af900.png)

For what it's worth, this is not urgent. While I don't see us abandoning the store app anytime soon, it's also not the primary installer, so it doesn't have to be on the front page yet. But I wanted to get the issue in so we don't forget it.

**Comment by berkerpeksag** [comment]
This should be available on staging.python.org soon. Please test it. I went with the second link, because our UA sniffing ability is pretty basic. I'm planning to rework all downloads application to have better support for OS detection and ability to find files for each Python release.

**Comment by zooba** [comment]
@berkerpeksag I don't see it yet - how soon is "soon"?

![image](https://user-images.githubusercontent.com/1693688/55810908-2219eb00-5a9d-11e9-81de-f6d46a898380.png)

**Comment by zooba** [comment]
Maybe it's just a limitation of the staging site? It looks even more broken on Firefox, and I don't have Chrome handy to test it there.

**Comment by berkerpeksag** [comment]
It's probably because of data on the staging site and cache invalidation. I will look into it.

Issues with Firefox might be a duplicate of https://github.com/python/pythondotorg/issues/1371.

**Comment by berkerpeksag** [comment]
You can now check the staging site. Note that downloads data is old :)

Screenshot:

<img width="403" alt="Screen Shot 2019-04-09 at 8 24 20 PM" src="https://user-images.githubusercontent.com/26338/55821252-b750ca00-5b05-11e9-8472-4c5f799c8f42.png">

**Comment by zooba** [comment]
Looks good in Edge. Firefox is suffering from that other issue.

**Comment by berkerpeksag** [comment]
Great! I will deploy this tomorrow night and invalidate caches on Fastly's side.

**Comment by ejodlowska** [comment]
Hi @berkerpeksag @zooba   - 

This change provides an unfair branding to one sponsor on the python.org site that is not part of any sponsor benefit we currently[1] provide.

I am not sure if this change was approved by someone, but I would like us to revert it back to include only the standard download button.  If there is more context here that I am missing, feel free to let me know via email. 

[1]we will discuss if branding like this _can_ be part of future sponsor benefits.

**Comment by ewdurbin** [comment]
If the logo is the primary issue, it may make sense to just add a note and link that Python is available in the MS store. There are certainly advantages to that installation path as it takes care of one of the biggest headaches with adding the python interpreter and tools to the users PATH and puts them on an upgradable installation course.

The only caveat here is that we do not provide anything similar to other Operating Systems or distributions (paid or otherwise), which arguably provide similar benefits.

For now we'll revert the Pull Request and reopen this issue for discussion.

**Comment by ewdurbin** [comment]
Revert: https://github.com/python/pythondotorg/commit/a31b422576e5572a07994eac9ca5bc7ed5b56bc1

**Comment by zooba** [comment]
I did very briefly wonder about why the badge isn't clear about the ownership of the package (unlike say the Google Play badge), but I also know that decisions at this level this aren't made without seriously considering the implications, so figured that following the official link guidelines was correct.

I'm perfectly happy with a plain text link, but as Ernest says, it's a much happier path for people landing on python.org these days so I'd like to figure out how to keep it there.

**Comment by ejodlowska** [comment]
We need to connect with a non-profit lawyer or two on this as well. As a non-profit, we shouldn't be linking to sponsor product pages. I wonder if this also applies here. We will need to confirm.

**Comment by zooba** [comment]
The page that's linked to is the product page that belongs to "us" (as in, you, Ernest and I have access, but nobody else at MSFT). The package is the one completely managed by volunteers, it's just the wording on the badge that MSFT says to use that isn't clear about that.

**Comment by ejodlowska** [comment]
Is this the link it went to: https://www.microsoft.com/store/apps/9NJ46SX7X90P?ocid=badge? It lives in the MSFT store.

**Comment by zooba** [comment]
Yep, that's the one. Remember that whole "verifying the PSF" thing we went through last year? That was for this page

**Comment by ejodlowska** [comment]
Sure, I recall.

The lines are blurry a bit as to what we can link to from our site and what can get us in trouble. We need to get confirmation.

**Comment by zooba** [comment]
Okay, that's fine. Just be clear when providing the information that we're linking to our own package that's being hosted on another site, and not linking to something that Microsoft has produced or claims any ownership of.

**Comment by saschanaz** [comment]
Would a plaintext that links to `ms-windows-store://pdp/?productid=9P7QFQMJRFP7` workaround the sponsor product issue? That one does not open Microsoft webpage and directly opens Microsoft Store app.

(The MSStore distribution not being in python.org makes some people think it's not official.)

**Comment by saschanaz** [comment]
It's not "completed" though, no? Did you intend to do "not planned"?

**Comment by JacobCoffee** [comment]
The current implementation of Python Install Manager is preferred according to the official CPython docs `Using Python on Windows`

**Comment by saschanaz** [comment]
That indeed sounds like not planned 👀

**Comment by hugovk** [comment]
Don't worry about the close state, it's not that important :)

**Comment by saschanaz** [comment]
It does matter when it's closed without any explanation 😛
